datasetinsights/commands/evaluate.py

```diff
@@ -0,0 +1,91 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Start model evaluation tasks for a pre-trained model.",
+    context_settings=const.CONTEXT_SETTINGS,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.STRING,
+    required=True,
+    help="Path to the config estimator yaml file.",
+)
+@click.option(
+    "-p",
+    "--checkpoint-file",
+    type=click.STRING,
+    required=True,
+    help="URI to a checkpoint file.",
+)
+@click.option(
+    "-d",
+    "--data-root",
+    type=click.STRING,
```

Comment:
Can be changed to `click.Path(exists=True)`

Reply:
fixed.